scripts/sources.seed.sh

```diff
@@ -4,7 +4,44 @@
 #
 BASEDIR=$(dirname $0)
 source $BASEDIR/sources.conf
-git clone https://github.com/MikelAlejoBR/sources-database-populator $BASEDIR/sources-database-populator
-pushd $BASEDIR/sources-database-populator
-go run main.go
-popd
+
+if [[ -z "$ARN_ROLE" ]]; then
+	echo "ARN_ROLE must be defined in sources.local.conf!"
+	exit 1
+fi
+
+ITITY=$($BASEDIR/identity_header.sh $ACCOUNT_ID $ORG_ID)
+
+echo "Creating $ARN_ROLE with account_id $ACCOUNT_ID org_id $ORG_ID"
+# create source for provisioning type with account 13/00013
+curl --location -g  --request POST "http://localhost:$PORT/api/sources/v3.1/bulk_create" \
+--header "X-RH-Identity: $ITITY" \
+-d '{
+	"sources": [
+		{
+			"name": "Amazon source",
+			"source_type_name": "amazon",
+			"app_creation_workflow": "manual_configuration"
+		}
+	],
+	"applications": [
+		{
+			"source_name": "Amazon source",
+			"application_type_name": "provisionin"
+		}
+	],
+	"authentications": [
+		{
+			"resource_type": "Application",
+			"resource_name": "provisioning",
+			"username": "$ARN_ROLE",
```

Comment:
isn't this enclosed in simple quotes for the -d `'` and in that case this variable would not be evaluated, right? :thinking:

Reply:
Ah good catch.